docs/docs/08.1-more-about-refs.md

```diff
@@ -68,7 +68,11 @@ Note that when the referenced component is unmounted and whenever the ref change
 
 ## The ref String Attribute
 
-React also supports using a string (instead of a callback) as a ref prop on any component, although this approach is mostly legacy at this point.
+> Note:
+>
+> Although string refs are not deprecated, they are considered legacy, and will likely be deprecated at some point in the future. Callback refs are preferred.
+
+React also supports using a string (instead of a callback) as a ref prop on any component.
```

Comment:
We generally use the term `prop` to refer to anything that is passed into an element (eg. using jsx).  I think it's probably fine as-is.